readme.html

```diff
@@ -0,0 +1,4 @@
+<h3>Note:</h3>
+<ul>
+    <li>Artifacts ingested during 'on poll' are changed and data paths for 'list sites' action are also changed</li>
```

Comment:
For version 3.0.0, Artifacts ingested during 'on poll' are changed and data paths for 'list sites' action are also changed as per the API changes so It is recommended to users update playbooks accordingly.

Reply:
updated